src/test/scala/com/amazon/deequ/VerificationSuiteTest.scala

```diff
@@ -374,11 +371,11 @@ class VerificationSuiteTest extends WordSpec with Matchers with SparkContextSpec
 
       // filtered rows 1, 2, 3 (where item > 3)
       val minRowLevel = resultData.select(expectedColumn4).collect().map(r => r.getAs[Any](0))
-      assert(Seq(true, true, true, true, true, true).sameElements(minRowLevel))
+      assert(Seq(null, null, null, true, true, true).sameElements(minRowLevel))
```

Comment:
these test were written with the intention that without specifying analyzer options, the default behavior would be filtered rows are `true` - related to above comment.

Reply:
Reverted the change.